utils/misc.mli

```diff
@@ -456,6 +456,7 @@ val print_if :
   Format.formatter -> bool ref -> (Format.formatter -> 'a -> unit) -> 'a -> 'a
 (** [print_if ppf flag fmt x] prints [x] with [fmt] on [ppf] if [b] is true. *)
 
+val suffix : int -> string
```

Comment:
Since `suffix` is only used when printing english ordinal numbers, you should probably export an ordinal_number: `Format.formatter -> int -> unit`  rather than `suffix`.